pySDC/projects/Resilience/fault_stats.py

```diff
@@ -212,14 +218,19 @@ def get_custom_description(self, problem, num_procs):
         elif problem == run_Schroedinger:
             e_tol = 4e-6
             dt_min = 1e-3
+        elif problem == run_leaky_superconductor:
```

Comment:
Looks like you're not testing the leaky superconductor, but changed the parameter in `fault_stats.py` to this (see below)?